adal/src/main/java/com/microsoft/aad/adal/AuthenticationException.java

```diff
@@ -33,6 +39,12 @@
 
     private ADALError mCode;
 
+    private HashMap<String, String> mHttpResponseBody = null;
```

Comment:
Can we change this left-side to Map<String, String>? (Coding to interface instead of concrete class)

Reply:
We have to use HashMap instead of Map. Because AuthenticationException is Serializable.